OMCompiler/Compiler/BackEnd/SymbolicJacobian.mo

```diff
@@ -4442,18 +4442,21 @@ algorithm
       updateList_arr := i_arr :: updateList_arr;
     end if;
   end for;
-    /* update adjacency matrix and transposed adjacency matrix */
+    /*
+      update adjacency matrix and transposed adjacency matrix
+      isInitial should always be false
+    */
     if not listEmpty(updateList_arr) then
       try
         /* scalar = true */
         SOME((mapEqnIncRow, mapIncRowEqn, indexType, true, _)) := syst.mapping;
-        syst := BackendDAEUtil.updateIncidenceMatrixScalar(syst, indexType, NONE(), updateList_arr, mapEqnIncRow, mapIncRowEqn);
+        syst := BackendDAEUtil.updateIncidenceMatrixScalar(syst, indexType, NONE(), updateList_arr, mapEqnIncRow, mapIncRowEqn, false);
```

Comment:
Might be good to have an isNotInitial constant here instead of just passing false

Reply:
For readability? I mean ... just a constant stating `Boolean isNotInitial = false`? 

I would not agree on that.

Reply:
Correct! if you have many parameters it is sometimes hard to know what a certain parameter is for. I usually do this for flags (or add a /**/ inline comment)

Reply:
I tried to cover that with the comment above "isInitial should always be false". But maybe that is not unambiguous enough.

Reply:
Oh my MetaModelica is rusty, do we need to pass parameters if they have a default value?

Reply:
We don't need to pass it if it has a default

Reply:
That is correct, but i don't want to give it a default value because that information could be crucial. I don't want that function to be used without thinking about providing this information. Usually you generate it from the shared object but sometimes you just know that it is not the initialization system, like in this case. 

I just want to prevent (mostly myself) from using this incorrectly.